libkbfs/journal_md_ops.go

```diff
@@ -345,22 +319,10 @@ func (j journalMDOps) GetUnmergedRange(
 
 func (j journalMDOps) Put(ctx context.Context, rmd *RootMetadata) (
 	MdID, error) {
-	bundle, ok := j.jServer.getBundle(rmd.TlfID())
+	tlfJournal, ok := j.jServer.getTLFJournal(rmd.TlfID())
```

Comment:
Same name as type, but that's a rule we break everywhere so feel free to leave it if you want to.